src/TextInputWithTokens.tsx

```diff
@@ -45,6 +45,10 @@ type TextInputWithTokensInternalProps<TokenComponentType extends AnyReactCompone
    * The size of the tokens
    */
   size?: TokenSizeKeys
+  /**
+   * The size of the text input
+   */
+  inputSize?: StyledWrapperProps['size']
```

Comment:
Curious about this one, could you provide some context on it please? I'm a bit confused about relationship to native `size` attribute on `<input`, which is a number? When I tried `small` or `large` it doesn't seem to make any noticeable difference either.

Reply:
Also think that given the component it named as a `**TextInput**WithTokens`, `size` prop should perhaps relate to input instead of tokens

Reply:
I agree, `size` should refer to the input. When this component was made, we used `variant` for size.

I'll change it and update `size` to be `tokenSize`.

Reply:
Or should this be a separate issue? It would technically be a breaking change

Reply:
This won't be an issue if we just remove `inputSize` as I'm recommending in https://github.com/primer/react/pull/1831#discussion_r802156523